### ISSUE TITLE
Installation on QGIS 3.22 / Ubuntu 20.04 throws `AttributeError: type object 'Qgis' has no attribute 'None' `

### ISSUE THREAD

**Issue by pka** [heremaps/xyz-qgis-plugin#50]
Couldn't load plugin 'XYZHubConnector' due to an error when calling its classFactory() method 

```
AttributeError: type object 'Qgis' has no attribute 'None' 
```
```
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/qgis/utils.py", line 423, in _startPlugin
    plugins[packageName] = package.classFactory(iface)
  File "/home/pi/.local/share/QGIS/QGIS3/profiles/default/python/plugins/XYZHubConnector/__init__.py", line 22, in classFactory
    from .plugin import XYZHubConnector
  File "/usr/lib/python3/dist-packages/qgis/utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/home/pi/.local/share/QGIS/QGIS3/profiles/default/python/plugins/XYZHubConnector/plugin.py", line 24, in 
    from .gui.space_dialog import MainDialog
  File "/usr/lib/python3/dist-packages/qgis/utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/home/pi/.local/share/QGIS/QGIS3/profiles/default/python/plugins/XYZHubConnector/gui/space_dialog.py", line 15, in 
    from ..xyz_qgis.controller import make_qt_args
  File "/usr/lib/python3/dist-packages/qgis/utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/home/pi/.local/share/QGIS/QGIS3/profiles/default/python/plugins/XYZHubConnector/xyz_qgis/controller/__init__.py", line 13, in 
    from ..common.signal import (BasicSignal, make_qt_args, output_to_qt_args,
  File "/usr/lib/python3/dist-packages/qgis/utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/home/pi/.local/share/QGIS/QGIS3/profiles/default/python/plugins/XYZHubConnector/xyz_qgis/common/signal.py", line 47, in 
    qgis_level = dict(zip(map(lambda k: getattr(Qgis,k), level), level))
  File "/home/pi/.local/share/QGIS/QGIS3/profiles/default/python/plugins/XYZHubConnector/xyz_qgis/common/signal.py", line 47, in 
    qgis_level = dict(zip(map(lambda k: getattr(Qgis,k), level), level))
AttributeError: type object 'Qgis' has no attribute 'None'
```

```
Python version: 3.8.10 (default, Sep 28 2021, 16:10:42) [GCC 9.3.0] 
QGIS version: 3.22.0-Białowieża Białowieża, d9022691f1 
```

Tested with version 1.8.3 from hub.qgis.org and 1.9.0 from Github releases.

**Comment by minff** [comment]
Hi @pka, this is indeed an issue in 1.8.3, but it has been resolved since experimental version 1.8.6 and also stable version 1.9.0. Could you try to uninstall/disable plugin version 1.8.3 before install version 1.9.0 ?

**Comment by pka** [comment]
You're right. After unnstallation of 1.8.3 and manual installation of 1.9.0 the plugin works. Thanks!
